source/guides/routing/overview.md

```diff
@@ -76,7 +76,16 @@ get '/rack-app', to: 'rack_app' # it will map to RackApp.new
 
 ### Mounting Applications
 
-If we want to mount an application, we should use `mount`.
+If we want to mount an application, we should use `mount` within Hanami environment configuration file. The default configuration file is located at `config/environment.rb`. Place `mount` within the Hanami.configure block.
```

Comment:
@spaceghoul don't feel dumb! it's a silly arcane thing, and now you know about it :)